apps/team-awesome-web/src/components/forms/select/Select.tsx

```diff
@@ -27,6 +27,7 @@ export const Select = ({ sortList, direction, setSortedResults, sortArray }: Sel
       ["Date", "startTimestamp"],
     ],
     teamMembers: [
+      ["Show all", "show all"],
```

Comment:
It seems a bit strange that "show all" is not in camel case here since all other values are, are you sure this is correct?

Reply:
Good point, changed for consistency